public/sitemap.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset
+        xmlns="http://www.sitemaps.org/schemas/sitemap/0.9"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.sitemaps.org/schemas/sitemap/0.9
+            http://www.sitemaps.org/schemas/sitemap/0.9/sitemap.xsd">
+
+    <url>
+        <loc>https://www.blumilk.pl/</loc>
+        <lastmod>2021-02-05T08:30:44+00:00</lastmod>
+    </url>
+
+    <url>
+        <loc>https://www.blumilk.pl/about</loc>
+        <lastmod>2021-02-05T08:30:44+00:00</lastmod>
```

Comment:
It would be nice to put here some real data based on last commits timestamps, but it's out of the range of this task. We should create a new one for CI flows with that.